app/services/publisher_wallet_setter.rb

```diff
@@ -2,32 +2,28 @@
 class PublisherWalletSetter < BaseApiClient
```

Comment:
Good point - I will correct the comment at the top. We should consider renaming the eyeshade endpoint as well if the name "wallet" is no longer applicable.